lib/hypernova/batch_renderer.rb

```diff
@@ -34,10 +37,11 @@ def initialize(jobs)
   #   "IonCannon.js" => <p>Feel my power!</p>,
   # }
   def render(response)
-    response.each_with_object({}) do |array, hash|
+    fmt_response = response.each_with_object({}) do |array, hash|
       name_of_component = array[0]
       hash[name_of_component] = extract_html_from_result(name_of_component, array[1])
     end
+    after_response(fmt_response, response)
   end
```

Comment:
is this intending to no longer return `fmt_response`, but instead, the result of `after_response`?